derive_builder_core/src/builder.rs

```diff
@@ -80,7 +83,7 @@ impl<'a> ToTokens for Builder<'a> {
                    impl_generics);
 
             tokens.append(quote!(
-                #[derive(Default, Clone)]
+                #[derive(Default, Clone #( , #derives)* )]
```

Comment:
TL;DR: Ok,let's just keep it as is :-)

My line of thinking was this:
- `derive_builder_core` is just a toolkit for code generation and should make minimal assumptions
- `derive_builder` populates the settings and sets all defaults - i.e. most decisions should be here

_But_ I just realized that `derive_builder_core` assumes the builder to implement `Clone` for immutable/mutable setters. So we already know in the scope of `derive_builder_core` that we need `Clone` for consistency. With this minimal reasoning rule of thumb the decision about deriving `Clone` should in fact stay in `derive_builder_core`. Now if only had `Default`, we could move at least that to `derive_builder`. But since we have both - let's just keep it as is. :-)